tests/release/const.go

```diff
@@ -59,6 +59,12 @@ const (
 )
 
 var paramsReleaseStrategy = []appstudiov1alpha1.Params{
+	{Name: "extraConfigGitUrl", Value: "https://github.com/scoheb/strategy-configs.git"},
```

Comment:
weren't we asked not to use use repos in the e2e tests?

Reply:
you are right, 
will fix it on https://issues.redhat.com/browse/HACBS-1854